skrifa/src/outline/autohint/instance.rs

```diff
@@ -30,3 +42,134 @@ impl GlyphStyles {
         }
     }
 }
+
+#[derive(Clone)]
+pub(crate) struct Instance {
+    styles: GlyphStyles,
+    metrics: UnscaledStyleMetricsSet,
+    target: Target,
+    is_fixed_width: bool,
+    style: Style,
+}
+
+impl Instance {
+    pub fn new(
+        font: &FontRef,
+        outlines: &OutlineGlyphCollection,
+        coords: &[F2Dot14],
+        target: Target,
+        styles: Option<GlyphStyles>,
+        lazy_metrics: bool,
```

Comment:
Doesn't have to be in this CL, but perhaps this could be changed from a raw bool to something more expressive? But it looks like this isn't public API, so I have no strong opinion on this.

Reply:
Yeah, this option isn’t actually exposed yet so you get lazy metrics when compiling with `std` and precomputed metrics otherwise (since without `std` we don’t have access to the synchronization primitives). I’m not really sure yet how to present this option to the user.

Reply:
Ack, that's fine with me.